discord/http.py

```diff
@@ -33,8 +33,8 @@
 from urllib.parse import quote as _uriquote
 
 import aiohttp
+import utils
```

Comment:
Although this works (I assume you tested, I didn't) I see circular imports coming in the future (see https://github.com/Pycord-Development/pycord/pull/2704 as an example)